Makefile

```diff
@@ -19,6 +19,7 @@ fmt:
 	black ./dependency_management/compare_fedora_versions
 	black ./dependency_management/set_lower_bounds
 	black ./misc_scripts/*
+	shfmt -l -w .
```

Comment:
Need a similar, but not necessarily identical entry in fmt-travis target.

Reply:
If I follow the convention of the commands in the fmt-travis, it would run the formatter, but in a way that would only check.

The `-d` switch prints a diff of what would have been modified, and returns an error code of 1 if there is a diff.

The `-l` switch lists the files that would be changed, but does not return an error if anything needs to be changed.

I'd prefer the command to return an error if anything needs to be changed.  While the diff is verbose, it seems to be the only option that returns the error for necessary changes.

Reply:
Ok! Looks like "-d" is the best choice.

Reply:
Resolved.